doc/conf.py

```diff
@@ -158,7 +159,7 @@ def _check_dependencies():
         'numpy': 'https://docs.scipy.org/doc/numpy/',
         'scipy': 'https://docs.scipy.org/doc/scipy/reference/',
     },
-    'backreferences_dir': 'api/_as_gen',
+    'backreferences_dir': Path('api') / Path('_as_gen'),
```

Comment:
as a side-comment, you don't actually need the second `Path` as they define "division" on include str on the right side.

Reply:
You can also write `Path('api', '_as_gen')` or (my preference) `Path('api/_as_gen')` (pathlib will normalize the slash to a backslash for you).